share-encryption/README.md

```diff
@@ -0,0 +1,42 @@
+# Encrypting configuration for share
+You can encrypt sensitive properties from share-custom-config.xml
+
+1. Run the Alfresco Share Encryption String Tool 
+    a.Navigate to https://github.com/Alfresco/share/share-encryption
+    b.Build the project using maven: mvn clean install
```

Comment:
Let's think about it and discuss whether this is the right approach (advising to build the tool from sources). Perhaps we should have the tool prebuilt and included in the Share distribution ZIP and Docker image.